src/variables.css

```diff
@@ -0,0 +1,7 @@
+:root {
+  --color-black: #060b35;
+  --color-grey: #616070;
+  --color-blue: #00acdc;
+  --color-white: #fff;
+  --color-slightly-dark-white: #f3f3f3;
```

Comment:
if you use some value only once, you shouldn't move it to the variable

Reply:
Removed and done